rust/pact_matching/src/query.rs

```diff
@@ -40,7 +42,7 @@ pub(crate) fn match_query_maps(
       None => result.entry(key.clone()).or_default().push(Mismatch::QueryMismatch {
         parameter: key.clone(),
         expected: "".to_string(),
-        actual: format!("{:?}", value),
+        actual: format!("{:?}", value.iter().map(|v| v.clone().unwrap_or_default()).collect_vec()),
```

Comment:
For debugging, it probably would be best to distinguish `Some("")` from `None`.

Reply:
Just for display purposes, keeps previous behavior. The expected and actual values are not actually used by anything.